portfolio/src/main/webapp/index.html

```diff
@@ -17,8 +17,17 @@
         <div id="popup-2" class="popup">
             popup2
         </div>
+        <!-- below is the contact popup -->
         <div id="popup-3" class="popup">
-            popup3
+            <p>To contact me just write an email or call</p>
+            <div class="contact-container">
+                <i class="material-icons md-36">email</i>
+                <p>andrzejpodobinski2@gmail.com</p>
+            </div>
+            <div class="contact-container">
+                <i class="material-icons md-36">call</i>
+                <p>+48 666 502 713</p>
```

Comment:
Generally it is not a good idea to include phone numbers in public :)

Reply:
yes, good advice, removed.